### PR TITLE
add: mama mia let me go

### PR DESCRIPTION
**Pull request by mgxd**

### CODE REVIEW

File: ME-MYSELF-I/Weekly-meetings.md [hunk 1]
```diff
@@ -1,5 +1,48 @@
...
+* Working branch of heudiconv - just needs testing updates
+
+#### What did I struggle with?
+
+* Unable to come to a conclusion on the task modeling - we should set up meeting with satra
+* Voice consents since redcap was down
+
+#### What would I like to work on next week?
+
+* Brain Plotting enhancements now that nibabel 2.2 is out
```
Comment (satra) [inline ME-MYSELF-I/Weekly-meetings.md:72]: take a look at niwidgets as well.

File: ME-MYSELF-I/Weekly-meetings.md [hunk 2]
```diff
@@ -1,5 +1,48 @@
...
+* Voice consents/scans
```
Comment (satra) [inline ME-MYSELF-I/Weekly-meetings.md:73]: how are we doing on remaining control scans?

**Comment by satra** [comment]
please do remember to add the weekly updates.

File: ME-MYSELF-I/Weekly-meetings.md (hunk 2 above)
Comment (mgxd) [reply ME-MYSELF-I/Weekly-meetings.md:73]: we're at 5 left, with one potential booked to consent

**Comment by mgxd** [comment]
@satra - whoops, committed last weeks as well, feel free to comment here

**Pull request merged**